### PR TITLE
srcrpm: builder [0/1]

### PR DESCRIPTION
**Pull request by fepitre**
Here is the part for making archive.

### CODE REVIEW

**Comment by marmarek** [comment]
Since you finished this part, can you drop "WIP" from the first commit message? Probably the best thing would be to squash first two commits, or even all three.

**Comment by fepitre** [comment]
Ok I do it. Tomorrow morning I will open PR on almost all the packages. Also, if you accept the two PR on the builders before, it will help to check for all the versions and releases.

File: scripts/create-archive [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# Define SOURCE_DATE_EPOCH from git latest commit timestamp
+SOURCE_DATE_EPOCH=$(git log -1 --format=%ct)
+
+# Create the archive:
+# - based on https://reproducible-builds.org/docs/archives/
+# - excluding .git, pkgs folder and prevent probable not so
+#   clever implementation of 'tar' which would result in
+#   an infinity loop due to tar '.'
+tar --sort=name \
```
Comment (marmarek) [inline scripts/create-archive]: Tar on travis (Ubuntu 14.04) don't support --sort=name :/
I see two options - use find | sort approach (see [this](https://reproducible-builds.org/docs/archives/#file-ordering)), or don't use file sorting on travis at all (making packages non-reproducible there). I'm slightly for the first option, but it means completely different archiving command for that environment.

**Comment by fepitre** [comment]
We can also handle the two cases e.g. testing tar version?

**Pull request merged**